src/Input/Select/Select.tsx

```diff
@@ -239,7 +239,11 @@ const Select: ISelect = (props: Props) => {
       className={classNames('aries-select', className)}
       ref={node}
     >
-      <SelectWrapper className="select-inputwrapper" isFocus={isFocus}>
+      <SelectWrapper
+        className="select-inputwrapper"
+        isFocus={isFocus}
+        small={small}
```

Comment:
Do you think using `size` to accept string ('small' | 'large') will be more scalable?

Reply:
I am only fix UI mismatching, don't want to change the interface in the same PR.